api/v1/ptpconfig_webhook.go

```diff
@@ -136,21 +139,18 @@ func (r *PtpConfig) ValidateDelete() error {
 	return nil
 }
 
-func getInterfaces(input *ptp4lConf, mode string) (interfaces []string) {
```

Comment:
If you are making multiple changes to the same line, please squash your commits.  It's very difficult to review code that's making changes to a commit that hasn't been reviewed already.

Reply:
squashed the commits